exceptional_operations.md

```diff
@@ -52,10 +52,16 @@ Note, however, that these are only recommendations. Uptane does not prescribe a
 
 ### Aftermarket ECUs
 
-A slightly more difficult use case to deal with are aftermarket ECUs -- for example, 3rd-party replacement parts, or add-on ECUs that add functionality for commercial fleet management. One approach is to work with the ECU manufacturer and treat them like any other tier-1 supplier. (The addition of the aftermarket ECU would be managed in one of the ways recommended in the previous subsection.) However, this may not be economically feasible in many cases. The easiest alternative is to simply exclude the aftermarket ECU from receiving OTA updates.
+A slightly more difficult use case to deal concerns the use of aftermarket ECUs -- for example, 3rd-party replacement parts, or add-on ECUs that add functionality for commercial fleet management. Though from a technical perspective adding an aftermarket ECU can be managed in one of the ways recommended in the previous subsection, there is no doubt that these components bring with them a set of unique logistical and security concerns. For starters, because aftermarket suppliers may not have access to the original design, these ECUs are often products of reverse engineering. As such, suppliers may not be able to glean all relevant design information about the ECU, including the rationale behind certain choices. Furthermore, the use of these components raises a number of fundamental questions, such as:
+- How do we deal with aftermarket ECUs that do not have their own Primary? Can they leverage an OEM's Director repository?
+- If an aftermarket ECU does have its own Primary, is each capable of controlling a mutually exclusive set of Secondaries?
+- Could ownership of a Director repository be delegated to a third party or owner?
```

Comment:
What does this mean?

Reply:
I understand this as: Could the owner (or a third party) direct the updates for their own vehicle by collecting both OEM and aftermarket updates?

Reply:
Again @mnm678 I am borrowing your verbiage. @trishankatdatadog does this clarify what was said?
```suggestion
- Could an owner (or a third-party) direct updates for their own vehicle from both an OEM and an aftermarket source?